src/app/home/home.page.html

```diff
@@ -25,29 +26,28 @@
       <ion-item-sliding *ngFor="let item of groupedCountries[letter]" (ionSwipe)="onItemDrag($event, item)">
         <ion-item-options side="start">
           <ion-item-option>
-            <ion-icon slot="icon-only" [name]="favourites.includes(item)? 'heart' : 'heart-outline'" color="success"></ion-icon>
+            <ion-icon slot="icon-only" [name]="favourites.includes(item)? 'heart' : 'heart-outline'"
+              color="success"></ion-icon>
             <span class="toggle-fav"> {{favourites.includes(item) ? 'Added!' : 'Not in favorite' }} </span>
           </ion-item-option>
         </ion-item-options>
 
         <ion-item-options side="end">
           <ion-item-option>
-            <ion-icon slot="icon-only" [name]="favourites.includes(item) ? 'trash-outline': 'trash'" color="danger"></ion-icon>
-            <span class="toggle-fav">{{favourites.includes(item) ? 'In favorite' : 'Removed!'}}   </span>
+            <ion-icon slot="icon-only" [name]="favourites.includes(item) ? 'trash-outline': 'trash'"
+              color="danger"></ion-icon>
+            <span class="toggle-fav">{{favourites.includes(item) ? 'In favorite' : 'Removed!'}} </span>
           </ion-item-option>
         </ion-item-options>
 
         <ion-item (click)="viewMore(item)">
           <ion-avatar slot="start">
             <ion-img [src]="item?.flag" (ionError)="onImageError($event)"></ion-img>
           </ion-avatar>
-          <div class="country-details">
-            <ion-label class="country-name"> {{item?.name}} </ion-label>
-            <ion-spinner *ngIf="isLoading && loadingItem === item" name="lines"></ion-spinner>
-            <span *ngIf="isLoaded && loadedItem === item">
-              : {{ item.population?.value === 0 ? 'Not Found' : item.population?.value?.toLocaleString() }}
-            </span>
-          </div>
+          <country-details [item]="item" [isLoading]="isLoading" [loadingItem]="loadingItem" [isLoaded]="isLoaded"
```

Comment:
use ion-infinite-scroll to lazy load item 
https://ionicframework.com/docs/api/infinite-scroll